module/VuFind/tests/unit-tests/src/VuFindTest/Resolver/Driver/AlmaTest.php

```diff
@@ -170,6 +173,96 @@ public function testParseLinks()
+                'coverage' => '',
+                'access' => 'limited',
+                'href' => 'https://na01.alma.exlibrisgroup.com/view/action/uresolver.do?operation=resolveService&package_service_id=5687861800000561&institutionId=561&customerId=550',
+                'notes' => '',
+                'authentication' => '',
+                'service_type' => 'getFullTxt',
+            ],
+            [
+                'title' => 'ebrary Science & Technology Subscription',
+                'coverage' => '',
+                'access' => 'limited',
+                'href' => 'https://na01.alma.exlibrisgroup.com/view/action/uresolver.do?operation=resolveService&package_service_id=5687861790000561&institutionId=561&customerId=550',
+                'notes' => '',
+                'authentication' => '',
+                'service_type' => 'getFullTxt',
+            ],
+            [
+                'title' => 'EBSCOhost Academic eBook Collection (North America)',
+                'coverage' => '',
+                'access' => 'open',
+                'href' => 'https://na01.alma.exlibrisgroup.com/view/action/uresolver.do?operation=resolveService&package_service_id=5687861770000561&institutionId=561&customerId=550',
+                'notes' => 'notessssssssssss SERVICE LEVEL PUBLIC NOTE',
+                'authentication' => 'collection level auth SERVICE LEVEL AUTHE NOTE',
+                'service_type' => 'getFullTxt',
+            ],
+            [
+                'title' => 'EBSCOhost eBook Community College Collection',
+                'coverage' => '',
+                'access' => 'limited',
+                'href' => 'https://na01.alma.exlibrisgroup.com/view/action/uresolver.do?operation=resolveService&package_service_id=5687861780000561&institutionId=561&customerId=550',
+                'notes' => '',
+                'authentication' => '',
+                'service_type' => 'getHolding',
+            ],
+            [
+                'title' => 'Elsevier ScienceDirect Books',
+                'coverage' => '',
+                'access' => 'limited',
+                'href' => 'https://na01.alma.exlibrisgroup.com/view/action/uresolver.do?operation=resolveService&package_service_id=5687861820000561&institutionId=561&customerId=550',
+                'notes' => '',
+                'authentication' => '',
+                'service_type' => 'getFullTxt',
+            ],
+            [
+                'title' => 'Request Assistance for this Resource!',
+                'coverage' => '',
+                'access' => '',
+                'href' => 'https://www.google.com/search?Testingrft.oclcnum=437189463&q=Fundamental+Data+Compression&rft.archive=9942811800561',
+                'notes' => '',
+                'authentication' => '',
+                'service_type' => 'getWebService',
+            ],
+        ];
+
+        $this->assertEquals($result, $testResult);
```

Comment:
In PHPUnit assertions, the first parameter is the "expected" value and the second is the "actual" value -- so I think these should be reversed.